Hooks/Base_Hook.php

```diff
@@ -57,7 +57,8 @@ final public function __callback() {
             /**
              * If we should not invoke callback, return the first function arg which is necessary for filters.
              */
-            return func_get_arg(0);
+            $func_args = func_get_args();
+            return count($func_args) ? func_get_arg(0) : '';
```

Comment:
@kilchdev Since we already have the func_args array, return $func_args[0] instead of making another function call to func_get_arg(). Probably not a big perf difference, but function calls are always more "expensive" that array index access.

Reply:
Agreed. Not sure why I didn't do that to start with...